rust/protocol/src/curve.rs

```diff
@@ -1,55 +1,80 @@
 //
-// Copyright 2020 Signal Messenger, LLC.
+// Copyright 2020-2021 Signal Messenger, LLC.
 // SPDX-License-Identifier: AGPL-3.0-only
 //
 
-mod curve25519;
+//! Identity creation primitives.
```

Comment:
Similarly (see identity_key.rs), these are keys but not necessarily identity keys.

Reply:
Tried to fix this! Let me know if the result works!